src/index.html

```diff
@@ -61,6 +61,7 @@
         </li>
     </ul>
     <button class="float-bottom btn btn-success" data-bind="click: exportAllData">Export complete dataset</button>
+    <button class="float-bottom btn btn-success" data-toggle="modal" data-target="#metadataModal">Show Metadata</button>
```

Comment:
It feels like maybe this button isn't in the right place, since it's showing metadata for a particular plot, but it's appearing in the sidebar, alongside 'Export complete dataset' as if it applies to all plots. Would it make more sense to put it next to the 'Export plot as..' buttons?

Could use bootstrap's margin classes to put a bit of space around these buttons, at the moment they're glued to the edges and each other.